user-script/popup/user-script.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
+
+const hostsInput =  document.querySelector("#hosts");
+const codeInput =  document.querySelector("#code");
+
+const defaultHosts = "*://*.org/*";
+const defaultCode = "document.body.innerHTML = '<h1>This page has been eaten<h1>'";
+
+hostsInput.value = defaultHosts;
+codeInput.value = defaultCode;
+
+async function registerScript() {
+  let background = await browser.runtime.getBackgroundPage();
```

Comment:
This won't run in private browsing mode, but just adding that to the readme might be enough.

Reply:
Hm, it would probably be better to use message-passing then. Nobody reads the docs.